device.go

```diff
@@ -496,7 +499,14 @@ func run() {
 		}
 
 		//Now send an uplink
-		ulfc, err := cDevice.Uplink(mqttClient, config.MQTT.UplinkTopic, config.Device.MType, uint8(config.RawPayload.FPort), &urx, &utx, payload, config.GW.MAC, config.Band.Name, dataRate, fOpts, fCtrl)
+		var ulfc uint32 = 0
```

Comment:
There's no need to assign `nil` to `err`. Also, prefer to declare `ulfc` like this:
```
var ulfc uint32
```